tests/js/spec/views/settings/incidentRules/create.spec.jsx

```diff
@@ -57,6 +57,7 @@ describe('Incident Rules Create', function() {
           project: [2],
           query: 'event.type:error',
           statsPeriod: '12h',
+          environment: [null],
```

Comment:
Is this expected?

Reply:
It was because a null environment was being passed in the case of "all environments". 

But I just made a change to remove that because that's not how discover is using this api. Noticed they were not passing any environment in that case. So I removed this now as well.